CHANGELOG.adoc

```diff
@@ -13,6 +13,10 @@ For a detailed view of what has changed, refer to the {url-repo}/commits/main[co
 
 == Unreleased
 
+Improvement::
+
+* Upgrade to asciidoctorj-diagram 2.2.1 (#1066)
```

Comment:
We could mention the issue https://github.com/asciidoctor/asciidoctorj/issues/1065, in case someone else stumbles upon the same problems.